Documentation/Architecture/InputSystem/CoreSystem.md

```diff
@@ -37,9 +33,6 @@ This object would get touch events when focused by a hand that is near it. These
 events go "up" the GameObject hierarchy until it finds a GameObject that is capable of handling
 the event.
 
-This is done by using [ExecuteHierarchy](https://docs.unity3d.com/ScriptReference/EventSystems.ExecuteEvents.ExecuteHierarchy.html)
```

Comment:
Might be good to leave the `ExecuteHierarchy` link in, to add more context on how / why this event system works the way it does.

Reply:
will restore. this is one instance where a link to the blob is most "reasonable"?

Reply:
done